app/code/Magento/Catalog/Model/Product/Attribute/Backend/TierPrice/UpdateHandler.php

```diff
@@ -129,12 +132,12 @@ private function getAdditionalFields(array $objectArray): array
      * Check whether price has percentage value.
      *
      * @param array $priceRow
-     * @return integer|null
+     * @return float|null
      */
-    private function getPercentage(array $priceRow)
+    private function getPercentage(array $priceRow): ?float
```

Comment:
Hi @SikailoISM, since Magento 2.2 supports PHP 7.0 you can't use nullable return types as this is a feature of PHP 7.1 and higher. See http://php.net/manual/en/migration71.new-features.php